nucypher/blockchain/eth/providers.py

```diff
@@ -90,10 +91,10 @@ def _get_auto_provider(provider_uri):
 
 def _get_pyevm_test_backend() -> PyEVMBackend:
     try:
+        # TODO: Consider packaged support of --dev mode with testerchain
         from tests.constants import PYEVM_GAS_LIMIT, NUMBER_OF_ETH_TEST_ACCOUNTS
     except ImportError:
-        raise ImportError("Nucypher is not installed in development mode.  "
-                          "Please checkout the code locally and reinstall to use the test client.")
+        raise DevelopmentInstallationRequired(importable_name='tests.constants')
```

Comment:
why `tests.constants`?

Reply:
Yeah - I suppose we are *really* trying to import `PYEVM_GAS_LIMIT`.

Reply:
:heavy_check_mark: